skills/lark-slides/scripts/xml_text_overlap_lint_test.py

```
@@ -2574,5 +2969,479 @@ def test_has_similar_short_card_peer_ignores_invisible_peers(self) -> None:
+              </data>
+            </slide>
+            """
+        )
+
+        self.assertEqual(issues, [])
+
+    def test_rejects_non_xsd_whitespace_in_color_pattern(self) -> None:
+        issues = self.validate(
+            f"""
+            <slide xmlns="{SML_NAMESPACE}">
+              <style><fill><fillColor color="rgb(1, 2,3)"/></fill></style>
```

Ruff (0.16.0)

[warning] String contains ambiguous ` ` (NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF001)